### PR TITLE
Submitting user data script updates

### PR DESCRIPTION
**Pull request by vikispeaks**

### CODE REVIEW

**Review by mjzac — APPROVED** [review]
The Script and Text sections seem to be merged together. Other than that, it looks good to me. @vikispeaks

File: building-web-forms/submitting-user-data/README.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+The server answers the request using the same protocol.
+
+The HTML form we create is a user-friendly way to configure an HTTP request to send data to a server. This enables the user to provide information to be delivered in the HTTP request.
+
+## Sending data
+
+The `form` element defines how the data will be sent to the web server. All of its attributes are designed to let you configure the request to be sent when a user clicks a submit button.
+
+The two most important attributes of a form element are `action` and `method`.
```
Comment (avishekjana) [inline building-web-forms/submitting-user-data/README.md:23]: The two most important attributes of a `form` element are `action` and `method`.

File: building-web-forms/submitting-user-data/README.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+### `action` attribute
+
+The `action` attribute defines where the data gets sent. Its value must be a valid URL. If this attribute isn't provided, the data will be sent to the URL of the page containing the form — the current page.
```
Comment (avishekjana) [inline building-web-forms/submitting-user-data/README.md:27]: The `action` attribute defines where we would like to send the data.

File: building-web-forms/submitting-user-data/README.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+```js
+
+   <form action="https://google.com"> #External URL
+   <form action="/localserver"> #Local URL
+
+```
+
+The names and values of the form controls are sent to the server as `name=value` pairs joined with the `&` symbol.
```
Comment (avishekjana) [inline building-web-forms/submitting-user-data/README.md:36]: An example for this statement would be helpful.

**Review by avishekjana — CHANGES_REQUESTED** [review]

**Comment by vikispeaks** [comment]
> The Script and Text sections seem to be merged together. Other than that, it looks good to me. @vikispeaks

@mjzac I am planning to keep this text only for most part. There might not be a script for video required.

File: building-web-forms/submitting-user-data/README.md (hunk 3 above)
Comment (vikispeaks) [reply building-web-forms/submitting-user-data/README.md:36]: The examples are stated below in the method related explanations.

File: building-web-forms/submitting-user-data/README.md (hunk 1 above)
Comment (vikispeaks) [reply building-web-forms/submitting-user-data/README.md:23]: Updated.

**Pull request merged**